src/AbstractPhpdocToTypeDeclarationFixer.php

```diff
@@ -208,6 +204,75 @@ protected function getCommonTypeFromAnnotation(Annotation $annotation, bool $isR
         return [$commonType, $isNullable];
     }
 
+    protected function getUnionTypes(TypeExpression $typesExpression, bool $isReturnType): ?string
+    {
+        if (\PHP_VERSION_ID < $this->versionSpecificTypes['union-types']) {
+            return null;
+        }
+
+        if (!$typesExpression->isUnionType() || '|' !== $typesExpression->getTypesGlue()) {
+            return null;
+        }
+
+        $types = $typesExpression->getTypes();
+        $isNullable = $typesExpression->allowsNull();
+
+        if (\count($types) < 2) {
+            return null;
+        }
+
+        $unionTypes = [];
+        $containsOtherThanIterableType = false;
+        $containsOtherThanEmptyType = false;
+
+        foreach ($types as $type) {
+            if ('' === $type) {
```

Comment:
how can a valid type be `''`?

Reply:
It was a quick fix for PHPStan because `empty()` was used here. I want to look at it more after #6356 is finished.